controllers/v1/task.go

```diff
@@ -20,6 +20,8 @@ type TaskController struct {
 func (ctrl TaskController) Routes(r *gin.RouterGroup) {
 	r.GET("/", ctrl.List)
 	r.GET("/:taskID/", ctrl.Get)
+	r.POST("/:taskID/mark-done", ctrl.MarkDone)
```

Comment:
keep single url, post for done, delete for un-done.
Also, should we call it resolve or done, chose which looks more apt to you

Reply:
Done